cartaya/card-2/index.html

```diff
@@ -8,18 +8,30 @@
     <title>card-2</title>
 </head>
 <body>
-    <div class="card-group_container">
-        <div class="card">
-            <img class="card_icon" src="./resources/icon/white.svg" alt="thunderbolt icon">
-            <p class="card_text">Your Top-up is received Al Instante</p>
+    <div class="card-group">
+        <div class="card card_one">
+            <div class="card_icon-container">
+                <img class="card_icon" src="./resources/icon/white.svg" alt="thunderbolt icon">
+            </div>
+            <div class="card_text-container">
+                <p class="card_text card_text-one">Your Top-up is received Al Instante</p>
```

Comment:
You don't need create a class to the text of each card. This is styles inerithed you can put it in the container